plugins/network-elements/juniper-contrail/src/main/java/org/apache/cloudstack/network/contrail/management/ServiceManagerImpl.java

```diff
@@ -134,7 +134,7 @@ private ServiceVirtualMachine createServiceVM(DataCenter zone, Account owner, Vi
         } catch (InsufficientCapacityException ex) {
             throw new CloudRuntimeException("Insufficient capacity", ex);
         }
-        CallContext.current().setEventDetails("Vm Id: " + svm.getId());
+        CallContext.current().setEventDetails("VM ID: " + svm.getId());
```

Comment:
```suggestion
        CallContext.current().setEventDetails("VM Id: " + svm.getId());
```

Reply:
@FelipeM525 , 'ID' is an acronym for 'identification' according to english rules it should be capatalised. Am I correct @jbampton ?

Reply:
@DaanHoogland 

I agree with capitalizing both letters, but, regardless of what convention we use (both capitalized or not), there should be a convention. Currently, this PR, that has a focus on fixing word casing inconsistencies, is doing the opposite, by changing some instances of `Id` to `ID` and leaving others as they are. Either all instances of the word in this PR should be `ID` or all should be `Id`.

Reply:
well, agree to disagree. I am happy to conform to conventions, but these are completely unnecessary and gratuit. I am leaving this with the embedded English speaker(s) ;)